src/lib/features/project/project-service.ts

```diff
@@ -1546,7 +1546,9 @@ export default class ProjectService {
             updatedAt: project.updatedAt,
             archivedAt: project.archivedAt,
             createdAt: project.createdAt,
-            onboardingStatus,
+            onboardingStatus: onboardingStatus ?? {
+                status: 'onboarding-started',
+            },
```

Comment:
This method already throws a not found error if the project id doesn't exist, so we can fall back to a default here. Is this the right default, though?